tests/cnf/ran/powermanagement/tests/cpufreq.go

```diff
@@ -24,11 +25,18 @@ var _ = Describe("CPU frequency tuning tests change the core frequencies of isol
 			desiredIsolatedCoreFreq = performancev2.CPUfrequency(2200002)
 			originalIsolatedCPUFreq performancev2.CPUfrequency
 			originalReservedCPUFreq performancev2.CPUfrequency
-			err                     error
 		)
 
 		BeforeEach(func() {
 
+			By("Checking that OCP version is 4.16 or later")
+			isGreaterOrEqual, err := rancluster.CompareOCPVersionWithCurrent(raninittools.Spoke1APIClient,
+				"4.18", true, false)
```

Comment:
Should this be 4.16?

Reply:
Fixed- that was a left-over from debugging. Thanks